server/src/main/scala/actors/daily/TerminalDayLikeActor.scala

```diff
@@ -81,11 +82,12 @@ abstract class TerminalDayLikeActor[VAL <: MinuteLike[VAL, INDEX], INDEX <: With
 
   def updateAndPersistDiff(container: MinutesContainer[VAL, INDEX]): Unit =
     diffFromMinutes(state, container.minutes) match {
-      case noDifferences if noDifferences.isEmpty => sender() ! MinutesContainer.empty[VAL, INDEX]
+      case noDifferences if noDifferences.isEmpty => sender() ! UpdatedMillis.empty
       case differences =>
         state = updateStateFromDiff(state, differences)
         val messageToPersist = containerToMessage(differences)
-        val replyToAndMessage = Option(sender(), MinutesContainer(differences))
+        val updatedMillis = if (container.contains(classOf[DeskRecMinute])) UpdatedMillis(differences.map(_.minute)) else UpdatedMillis.empty
```

Comment:
Could we extract `container.contains(classOf[DeskRecMinute])` to a function which conveys the intent here - it's not completely obvious to me what's going on.

Reply:
Yes good suggestion